src/Compilers/Core/Portable/SourceGenerator/SourceGeneratorExtensions.cs

```diff
@@ -16,7 +16,9 @@ public static class SourceGeneratorExtensions
             ImmutableArray<SourceGenerator> generators,
             string path,
             bool writeToDisk,
-            CancellationToken cancellationToken)
+            Func<Exception, bool> exceptionFilter,
```

Comment:
The `exceptionFilter` parameter was intended to match the corresponding parameter for `AnalyzerDriver`, but all callers are currently passing `null` so the parameter is not used or tested yet so I'll remove it.